src/lsdb_macauff/config/all_sky_params.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class AllSkyParams:
+    """Joint sky parameters"""
+
+    include_perturb_auf: bool
+    include_phot_like: bool
+    use_phot_priors: bool
+    cross_match_extent: List[float]
```

Comment:
Remove `cross_match_extent` here as well as in the `yml` file

Reply:
Thanks @Onoddil, just did!